ui/components/telemetry/grafana/GrafanaConfigComponent.js

```diff
@@ -56,7 +56,7 @@ class GrafanaConfigComponent extends Component {
                   margin="normal"
                   variant="outlined"
                   onKeyDown={(e) => {
-                    if (e.keyCode == 13) {
+                    if (e.key == 'Enter') {
```

Comment:
What if this `'if` statement is not true? Are we handling that case? If yes, then can you please point that out?  
As for better user experience. 
Also if we only have `if` statement, how about using `&&` operator?

Reply:
done.